gb2260/division.py

```diff
@@ -119,10 +119,8 @@ def stack(self):
 def make_year_key(year):
     """A key generator for sorting years."""
     if year is None:
-        return (LATEST_YEAR, 12)
+        return (LATEST_YEAR, 7)
     year = str(year)
-    if len(year) == 4:
-        return (int(year), 12)
```

Comment:
What is the reason you change here for?

The structure of 2-tuple should be `(year, month)`. The default month should be `12` for solo-revised years.